### ISSUE TITLE
Passing UUID to gnome-shell-extension-prefs no longer supported since 3.36.1

### ISSUE THREAD

**Issue by smcv** [EasyScreenCast/EasyScreenCast#247]
(For context: I don't use this extension myself, but I'm going through GNOME Shell extensions that are packaged in Debian and making sure there are issues open to get them adapted to an upstream change.)

This extension has:

```
        Main.Util.trySpawnCommandLine('gnome-shell-extension-prefs  EasyScreenCast@iacopodeenosee.gmail.com');
```

which will not work since GNOME Shell 3.36.1, because the way extension preferences work has been redone to accommodate the g-s-extension-prefs app being installed via Flatpak or similar.

As far as I can tell from GNOME Shell's upstream commit history, the preferred way to launch extension preferences in sufficiently recent versions is to call imports.misc.extensionUtils.openPrefs(), which was added by [gnome-shell!1163](https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1163). The version in Ubuntu 20.04 doesn't have that function available yet, but hopefully will soon; Debian experimental already has it.

To support older GNOME versions, fall back to the spawn call if imports.misc.extensionUtils doesn't have an openPrefs method.

Sample code: [Tudmotu/gnome-shell-extension-clipboard-indicator#203](https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/pull/203)

**Comment by tobias47n9e** [comment]
@smcv Thanks for the detailed report. The issue is fixed in version 1.2.0 and the new version should be published on https://extensions.gnome.org in a few days.

**Comment by adangel** [comment]
This has been fixed with 1.2.0
